src/set_theory/surreal/basic.lean

```diff
@@ -198,44 +192,16 @@ begin
   { exact numeric_one }
 end
 
-theorem half_add_half_equiv_one : half + half ≈ 1 :=
```

Comment:
Just to note: I did this because the proof no longer required any lemmas about `numeric`. The proof can still be golfed down **a lot**, but I'll do this at a later point.